test/expect-tests/dune_patch/dune_patch_tests.ml

```diff
@@ -58,6 +58,18 @@ index ea988f6bd..000000000
 |}
 ;;
 
+(* Use GNU diff 'unified' format instead of 'git diff' *)
```

Comment:
This should move to #10859 and then when it is rebased it should show that the code parses it properly now.

Reply:
The repro PR has been merged, and this PR has been rebased on that.